server/src/port/module/auth.module.ts

```diff
@@ -0,0 +1,37 @@
+import { Module } from '@nestjs/common';
+import { AuthService } from '../../app/auth/auth.service';
+import { PassportModule } from '@nestjs/passport';
+import { UserModule } from './user.module';
+import { JwtModule, JwtSecretRequestType, JwtService } from '@nestjs/jwt';
+import { UserServiceImpl } from '../../app/user/user.service.impl';
+import { ValidationServiceImpl } from '../../domain/user/validation.service.impl';
+import { UserRepository } from '../persistence/repository/user.repository.impl';
+import { Web3BridgeService } from '../adapter/service/blockchain/ethereum/web3/web3.bridge.service';
+import Web3 from 'web3';
+import { AuthController } from '../../web/auth.controller';
+import { JwtStrategy } from '../../app/auth/jwt.strategy';
+
+@Module({
+    imports: [
+        PassportModule.register({ defaultStrategy: 'jwt' }),
+        JwtModule.register({
+                secretOrKeyProvider: () => {
+                    return 'secretKey';
+                },
+            signOptions: {
+                expiresIn: '10h',
```

Comment:
access token expires in 3600.
why 10h is here?
is it different from access token?